ideprobe/tests/src/test/scala/com/twitter/intellij/pants/Suites.scala

```diff
@@ -0,0 +1,23 @@
+package com.twitter.intellij.pants
+
+import org.junit.runner.RunWith
+import org.junit.runners.Suite
+import org.junit.runners.Suite.SuiteClasses
+
+@RunWith(classOf[Suite])
+@SuiteClasses(Array(
+  classOf[OpenProjectTestBsp],
+  classOf[OpenProjectTestPants],
+  classOf[BUILDFilesTest],
+  classOf[PantsSettingsTest],
+  classOf[ThriftIdeaPluginTest]
+))
+class Suite1
```

Comment:
Could we use more descriptive names?

Reply:
These are only for sake of grouping. There is no logic behind it except for that both take similar time to complete

Reply:
So the grouping is completely arbitrary and those test classes have nothing in common? In that case we could call them batches and comment on how the grouping works, so the next person adding a test does not need to wonder where to include it.